### PR TITLE
updated datafile, added clarity to reference

### PR DESCRIPTION
**Pull request by mauerbac**
@haleybash-optimizely @vignesh-optimizely @delikat @jprovine 

I left a few comments below. Please pay close attention to the datafile section. The sample datafile should be linked now (rather than displaying on the right).

### CODE REVIEW

File: src/content/server/reference/datafile/0-intro.md [hunk 1]
```diff
@@ -2,195 +2,16 @@
 template: multi-example
 title: Datafile
 anchor: datafile
+datafile: true
```
Comment (mauerbac) [inline src/content/server/reference/datafile/0-intro.md]: After updating the datafile with all relevant components it became too larger to list in the side bar. I've linked to an example here instead. The button needs a bit of work from a designer to leverage OUI.

File: src/content/server/reference/init/0-intro.md [hunk 2]
```diff
@@ -49,45 +49,46 @@ code_examples:
 <div class="hidden visible" data-toggle-section="python-code">
 <p>First, you'll need to import the Optimizely library into your code as shown.</p>
 
-To run experiments with Optimizely you'll need to instantiate an `Optimizely` object in your code. This object represents the state of an Optimizely project and can be used to both [activate experiments](#activation) and [track events](#tracking).</div>
+<p>To run experiments with Optimizely you'll need to instantiate an `Optimizely` object in your code. This object represents the state of an Optimizely project and can be used to both [activate experiments](#activation) and [track events](#tracking).</p>
 
-<p>
+<p>To construct the `Optimizely` object, you'll need to provide the `datafile` as an argument representing the JSON configuration of your project. The datafile contains all the instructions needed to activate experiments and track events. For more information on the expected format of the datafile and how to get the most up-to-date version, see the [datafile](#datafile) section.</p>
```
Comment (mauerbac) [inline src/content/server/reference/init/0-intro.md]: Added link to custom config at the bottom

File: src/content/server/reference/tracking/0-intro.md [hunk 3]
```diff
@@ -39,4 +39,5 @@ code_examples:
 
 You can easily track Optimizely goals from your code as shown in the example to the right. Note that you don't need to pass the assigned experiments or variations; Optimizely will drop any conversion events for users that are not part of an experiment that includes the goal.
 
-<div class="attention attention--warning push--bottom">*Please note:* in cases where an experiment is activated for a user, the SDK will log an event to Optimizely servers to record the conversion event.</div>
```
Comment (mauerbac) [inline src/content/server/reference/tracking/0-intro.md]: This was confusing to me (I think this was only intended for the activate section). I've added a comment that's more relevant.

File: src/partials/_multi-example.html [hunk 4]
```diff
@@ -47,8 +47,27 @@ <h3 id="{{object.anchor}}">
 <!-- Indenting is weird here because of the <pre> tags -->
 
 <!-- Example Request -->
+{% if object.datafile %}
+<h4>Example</h4>
+
+<div style="font-size:20px;margin-top:25%;margin-left:25%">
+
+<form action="/data/datafile.html">
+    <button class="lego-button" type="submit"> View Sample Datafile </button>
```
Comment (mauerbac) [inline src/partials/_multi-example.html]: @danoc, could you help me use the OUI button here?  :)

File: src/partials/_multi-example.html [hunk 5]
```diff
@@ -50,10 +50,10 @@ <h3 id="{{object.anchor}}">
 {% if object.datafile %}
 <h4>Example</h4>
 
-<div style="font-size:20px;margin-top:25%;margin-left:25%">
+<div class="text--center" style="font-size:20px;margin-top:25%">
 
 <form action="/data/datafile.html">
-    <button class="lego-button" type="submit"> View Sample Datafile </button>
```
Comment (mauerbac) [inline src/partials/_multi-example.html]: fixed styling!

**Comment by vraja2** [comment]
when I click on "View sample datafile" I get "Cannot GET /data/datafile.html?"

**Pull request merged**